### PR TITLE
Add editor reset button

### LINKED ISSUE #629: Be able to restart editor
Sometimes you want to bring back to the editor the original default code. It would be nice to have a small button that clears the editor. 

We could use this icon: http://fontawesome.io/icon/trash-o/

### PR DESCRIPTION
**Pull request by flbulgarelli**
Fixes #629 

<del>
This is feature has to be discussed. I need it currently in order to develop exercises easier. it my be useful for student but I am not completly sure about it's actual need. 

</del>

### CODE REVIEW

**Comment by flbulgarelli** [comment]
It looks this way


reset

**Comment by flbulgarelli** [comment]
Perhaps we could rename `borrar` -> `reiniciar`, and also apply the same styles in console - which is good, since the `reiniciar` button is too big

**Comment by aguspina** [comment]
I really like your last idea!

**Comment by PalumboN** [comment]
I prefer `reiniciar`. And maybe [change the icon](http://fontawesome.io/icon/undo/)?

My two cents! :P

**Comment by aguspina** [comment]
@flbulgarelli please review and merge!

**Comment by flbulgarelli** [comment]
@aguspina could you provide any screenshot please?

File: spec/models/exercise_spec.rb [hunk 1]
```diff
@@ -148,11 +148,11 @@
     context 'when user has a single submission for the exercise' do
```
Comment (flbulgarelli) [inline spec/models/exercise_spec.rb]: We should rename the describe's name

**Comment by aguspina** [comment]
Is the same as yours previous imagen but with 'reset' text instead of
'delete'

File: app/views/layouts/submissions/editor/_problem_form.html.erb [hunk 2]
```diff
@@ -22,14 +22,17 @@
...
       <% if editor_mode == :visible %>
         <div class="field form-group">
           <%= f.editor :content, 'dynamic',
                        class: 'form-control editor',
-                       value: @default_content,
+                       value: @current_content,
                        style: 'display: none' %>
         </div>
+        <div class="editor-overlapped">
+          <a class="editor-trash"><%= fa_icon('trash-o', text: t(:reset))%></a>
```
Comment (flbulgarelli) [inline app/views/layouts/submissions/editor/_problem_form.html.erb]: perhaps we could rename this style to `editor-reset` to be consistent with names in model.

File: app/views/layouts/submissions/editor/_playground_form.html.erb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <div class="console">
 </div>
-<div class="pull-left">
-  <div class="btn btn-info clear-console"> <%= t :clear_console %></div>
+<div class="console-overlapped">
+  <a class="clear-console"><%= fa_icon('trash-o', text: t(:reset))%></a>
```
Comment (flbulgarelli) [inline app/views/layouts/submissions/editor/_playground_form.html.erb]: We should rename this class too - `console-reset`

Comment (flbulgarelli) [inline app/views/layouts/submissions/editor/_playground_form.html.erb]: Also please notice that this line

```
fa_icon('trash-o', text: t(:reset))
```

is duplicated. 

We could add a helper `reset_button`

File: app/assets/stylesheets/application/modules/_console.scss [hunk 4]
```diff
@@ -48,6 +48,17 @@ div.console span.jquery-console-prompt-label {
   color: #3498db;
 }
 
-div#console .clear-console {
+.clear-console {
   cursor: pointer;
 }
+
+.total-width {
```
Comment (flbulgarelli) [inline app/assets/stylesheets/application/modules/_console.scss]: Are you using this class? I didn't found any reference to it

File: spec/models/exercise_spec.rb (hunk 1 above)
Comment (aguspina) [inline spec/models/exercise_spec.rb]: @flbulgarelli what do you propose?

Comment (flbulgarelli) [inline spec/models/exercise_spec.rb]: current_content_for :P

Comment (flbulgarelli) [inline spec/models/exercise_spec.rb]: But never mind. Not a big issue. merging

**Pull request merged**